packages/@okta/vuepress-site/docs/reference/api/asa/audits/index.md

```diff
@@ -0,0 +1,129 @@
+---
+title: ASA Audit Events
+category: asa
+---
+
+# ASA Audit Events API
+
+## Get Started
+
+
+| Product  | API Basics  | API Namespace        |
+|----------|-------------|----------------------|
+| [Advanced Server Access](https://www.okta.com/products/advanced-server-access/) | [How the ASA API works](../introduction/) | `https://app.scaleft.com/v1/`
```

Comment:
[How the ASA API works](../introduction/) is not a well-formed link. All links should be relative to `developer.okta.com`